### PR TITLE
Removing old snackbar.

### PR DESCRIPTION
**Pull request by llvasconcellos**
Migration of error messages to new snackbar and removing the old one.
The messages don't pile up anymore!

### CODE REVIEW

File: app/src/main/java/org/projectbuendia/client/ui/BaseActivity.java [hunk 1]
```diff
@@ -132,6 +129,12 @@ public int snackBar(String message, String actionMessage, View.OnClickListener l
         return snackBar.message(message, actionMessage, listener, 999);
     }
 
+    public int snackBar(int resourceIdMessage, int resourceIdActionMessage, View.OnClickListener
```
Comment (capnfabs) [inline app/src/main/java/org/projectbuendia/client/ui/BaseActivity.java]: Use the `@StringRes` annotation for this and Android Studio will warn you if you accidentally pass the wrong type of value in here :)

**Comment by capnfabs** [comment]
I think I would do it at the snackbar level similarly to how [Android does notifications](http://developer.android.com/guide/topics/ui/notifiers/notifications.html#Updating). For notifications, you have to pass a unique numeric ID to the notification function, which defines whether it overwrites an existing notification or not. For snackbar, this means you'd have to re-architect some of the functions to accept an ID instead of generate an ID, but I think this is a pretty clean solution.

**Comment by dancunningham** [comment]
Not allowing duplicates sounds good to me, and if a notification is triggered a second time, it should bump up to the top of the list within its priority level.

**Comment by capnfabs** [comment]
@dancunningham just have to be careful then to assign different priorities to all the health check messages, lest they keep swapping positions arbitrarily.

**Pull request merged**